packages/vaex-ml/vaex/ml/tensorflow.py

```diff
@@ -0,0 +1,70 @@
+from functools import partial
+
+
+import tensorflow as tf
+
+import tensorflow_io.arrow as arrow_io
+from tensorflow_io.arrow.python.ops.arrow_dataset_ops import arrow_schema_to_tensor_types
+
+
+class DataFrameAccessorTensorflow(object):
+    def __init__(self, ml):
+        self.ml = ml
+        self.df = self.ml.df
+
+    def _arrow_batch_generator(self, features, target=None, chunk_size=1024):
+        column_names = features + [target] if target is not None else features
+        for i1, i2, table in self.df.to_arrow_table(column_names=column_names, chunk_size=chunk_size):
+            yield table.to_batches(chunk_size)[0]
+
+    @staticmethod
+    def _get_batch_arrow_schema(arrow_batch):
+        output_types, output_shapes = arrow_schema_to_tensor_types(arrow_batch.schema)
+        return output_types, output_shapes
+
+    def to_dataset(self, features, target=None, chunk_size=1024, repeat=None, shuffle=False, keras=False):
```

Comment:
I think keras should be called as_dict (inverting the Boolean logic)

Reply:
Yeah we could do that. I thought this way it is more obvious which option is for what. I guess we can have it as_dict as you suggest, and describe in the docstring what it is for?

Reply:
Now it is implemented `as_dict=True` (use to be `keras=False`)